.gitignore

```diff
@@ -9,3 +9,7 @@ pkg/*
 # YARD
 .yardoc
 yardoc/
+
```

Comment:
Can you please revert these changes? They are not relevant for the fix itself.

Reply:
Sure. It would be good if you readded it though.